WordPress/src/wordpress/java/org/wordpress/android/ui/accounts/login/compose/components/Tagline.kt

```diff
@@ -0,0 +1,72 @@
+package org.wordpress.android.ui.accounts.login.compose.components
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.wordpress.android.R.color
+import org.wordpress.android.R.drawable
+import org.wordpress.android.R.font
+import org.wordpress.android.R.string
+import org.wordpress.android.ui.compose.theme.AppTheme
+
+private val fontSize = 22.sp
+private val lineHeight = fontSize * 1.3
+
+@Composable
+fun ColumnScope.Tagline(text: String, modifier: Modifier = Modifier) {
+    Column(
+            modifier = modifier.weight(1f),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
+    ) {
+        Image(
+                painter = painterResource(drawable.ic_wordpress_gridicon),
+                contentDescription = stringResource(string.login_prologue_revamped_content_description_wordpress_icon),
+                colorFilter = ColorFilter.tint(colorResource(id = color.login_prologue_revamped_icon)),
+                modifier = Modifier
+                        .size(50.dp)
+        )
+        Text(
+                text,
+                Modifier
+                        .width(234.dp)
+                        .padding(top = 25.dp),
+                fontFamily = FontFamily(Font(font.eb_garamond)),
```

Comment:
This is looking good to me at the moment, though I have a feeling at some point in the near future we'll have to define a material typography theme.

Reply:
Yeah, I have deferred hoisting this, since this is so far the only use, but it certainly makes sense to make this more "global" when we have other uses. 👍

Reply:
Actually, I decided, since we seem to be planning more usage of this font, we may as well think forward to wrangle this here, since it's simple enough to do. I've extracted it in this commit: https://github.com/wordpress-mobile/WordPress-Android/pull/17104/commits/d41cad93366b719672c98ae1330cbc0fb39bc6f4

Reply:
Actually, I decided, since we seem to be planning more usage of this font, we may as well think forward to wrangle this here, since it's simple enough to do. I've extracted it in this commit: https://github.com/wordpress-mobile/WordPress-Android/pull/17104/commits/d41cad93366b719672c98ae1330cbc0fb39bc6f4